### ISSUE TITLE
channels of Decoder

### ISSUE THREAD

**Issue by hwd-code** [mapooon/BlendFace#13]
self.Decoder_inchannel = [1024, 2048, 1024, 512, 256, 128]
        self.Decoder_outchannel = [1024, 512, 256, 128, 64, 32]
        self.Decoder = nn.ModuleDict({f'layer_{i}' : nn.Sequential(
                nn.ConvTranspose2d(self.Decoder_inchannel[i], self.Decoder_outchannel[i], kernel_size=4, stride=2, padding=1),
                nn.BatchNorm2d(self.Decoder_outchannel[i]),
                nn.LeakyReLU(0.1)
            )for i in range(6)})

self.Decoder_inchannel[1](2048) is not equal to self.Decoder_outchannel[0](1024).is there a problem here
i think there is an issue with setting the number of channels.
